src/Kiota.Builder/Refiners/CommonLanguageRefiner.cs

```diff
@@ -386,7 +386,7 @@ protected static void ReplaceReservedNames(CodeElement current, IReservedNamesPr
         // 1. In the list of reserved names
         // 2. If it is a reserved name, make sure that the CodeElement type is worth replacing(not on the blocklist)
         // 3. There's not a very specific condition preventing from replacement
-        if (provider.ReservedNames.Contains(current.Name) &&
```

Comment:
Instead of impacting the contains here, can you change the init code please?
https://github.com/microsoft/kiota/blob/1f60de908075fd51eebdc257ead049cf498e225c/src/Kiota.Builder/Refiners/GoReservedNamesProvider.cs#L7

Add (StringComparer.OrdinalIgnoreCase) after new HashSet<String>

Reply:
Nice, done!